### PR TITLE
add path to intro notebook

### PR DESCRIPTION
**Pull request by orchid00**

### CODE REVIEW

File: slides/background_python_and_jupyter_notebook.html [hunk 1]
```diff
@@ -209,7 +209,7 @@
 
   - **Terminal** and a **text editor**
 
-      - We recommend [IPython](https://ipython.org/)
+      - We recommend [Jupyter notebook](https://jupyter.org/)
```
Comment (tclose) [inline slides/background_python_and_jupyter_notebook.html:212]: Jupyter notebooks are not technically "terminal and text editor" and they are introduced below. I would like to leave this as it was

File: slides/background_python_and_jupyter_notebook.html [hunk 2]
```diff
@@ -252,7 +252,7 @@
 
 ## Jupyter Notebook
 
-Please go through the [`Jupyter Notebook jupyter notebook`](../../notebooks/notebooks/introduction_jupyter-notebook.ipynb) now
+Please go to the folder `project/notebooks/` and type in the console [`jupyter notebook introduction_jupyter-notebook.ipynb`](../../notebooks/notebooks/introduction_jupyter-notebook.ipynb) now
```
Comment (tclose) [inline slides/background_python_and_jupyter_notebook.html:255]: They don't need to go to the console as they should be able to just click on the link.

File: slides/background_python_and_jupyter_notebook.html [hunk 3]
```diff
@@ -320,7 +320,7 @@
 
 ## Python Basics
 
-Please go through the [`Python basics jupyter notebook`](../../notebooks/notebooks/introduction_python.ipynb) now
+Please go through the [`jupyter notebook introduction_python.ipynb`](../../notebooks/notebooks/introduction_python.ipynb) now
```
Comment (tclose) [inline slides/background_python_and_jupyter_notebook.html:323]: The user won't need to know that the file is called 'introduction_python.ipynb' as they can just use the links here and on the program page

File: slides/background_python_and_jupyter_notebook.html [hunk 4]
```diff
@@ -383,7 +390,8 @@
 ```
 --
 ```python
-# Strict indentation (number of leading spaces v. important!)
+# Strict indentation 
+# number of leading spaces is very important!
```
Comment (tclose) [inline slides/background_python_and_jupyter_notebook.html:394]: I think I prefer how I had it before, what is your thinking behind this change?

File: slides/background_python_and_jupyter_notebook.html [hunk 5]
```diff
@@ -357,7 +357,14 @@
...
 'two', 'one'
 ```
---
+---
+      
+name: important1a
+class: left
+layout: false
+name: intro      
+## Python - Important to know 1
```
Comment (tclose) [inline slides/background_python_and_jupyter_notebook.html:366]: Should this be "Important to know 2"?

**Review by tclose — CHANGES_REQUESTED** [review]
Hi Paula,

I have made some comments on your proposed changes.

Cheers,

Tom

**Comment by orchid00** [comment]
@tclose I'll discard the changes,

**Pull request closed without merging**